app/settings.py

```diff
@@ -0,0 +1,147 @@
+"""
+Django settings for app project.
+
+Generated by 'django-admin startproject' using Django 2.0.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.0/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '5f^ouu_s%s@@(476pwmglzanj58a#eiv)p&v3x7h6_cs5+6@#c'
```

Comment:
I don't see references to this, do we need it? If so, get it from `os.environ` rather than hardcoding please.

Reply:
Django uses the secret key for making hashes. We do need it, as Django will refuse to start if SECRET_KEY is not set. And yes, I'm now fetching it from environment!